src/granite_orm/validators.cr

```diff
@@ -5,12 +5,23 @@ module Granite::ORM::Validators
 
   macro included
     macro inherited
+      @@validators = Array({field: Symbol, message: String, block: Proc(self, Bool)}).new
```

Comment:
Having both a `@@validators` and a `@validators` is confusing.

Reply:
Perhaps one syntax can wrap the other? This code is an untested wild guess, but:

```crystal
macro validate(field, message, &block)
  validate field, message, -> (model : Model) do
    {{ yield }}
  end
end
```

Reply:
I agree.  Plus it will bring back the original issue where you validate one object and all the other objects show as invalid as well.